relay-common/src/constants.rs

```diff
@@ -478,29 +478,32 @@ impl fmt::Display for FractionUnit {
     }
 }
 
+const CUSTOM_UNIT_SIZE: usize = 15;
```

Comment:
I'd note that the number is a max/threshold/limit. For example:
```suggestion
const CUSTOM_UNIT_MAX_SIZE: usize = 15;
```